### DISCUSSION TITLE
Webpack output bundle path

### DISCUSSION THREAD

**Discussion by CharlesJQuarra** [quasarframework/quasar#9457] [CLI - Capacitor / Cordova modes] [answered]
I've spent an afternoon trying to figure out how to detect where is the quasar/vue bundle being generated by webpack. I have a `src-cordova` folder as that is the only platform target I have enabled atm. I followed this: [https://quasar.dev/quasar-cli/handling-webpack](url) and did `quasar inspect -c dev -m cordova -d 17 > webpack.config`. According to [https://webpack.js.org/configuration/output/](url) I should expect an `output` node in the output with, among other options, the output bundle path. The generated config is large, but still couldn't find any hints of a path to an `output`, a `bundle.js` or anything that one might expect as the compiled bundle output.  I couldn't find any further documentation in the Quasar pages that would elucidate this mystery

**Comment by ontwikkelfabriek** [comment, marked as answer]
@CharlesJQuarra 

In Cordova mode the generated files that contain the Quasar code in a  are in the `myproject/src-cordova/www` folder. If you run a `build` command, for android the apk's are in the folder `myproject/dist/cordova/apk/release`  ( at least for Qv1)
